l10n/en/navigation.ftl

```diff
@@ -19,9 +19,6 @@ navigation-watch-videos-and-browse = Watch videos and browse the internet on you
 navigation-firefox-browsers-put = { -brand-name-firefox } browsers put your privacy first — and always have.
 navigation-take-the-passwords-youve = Take the passwords you’ve saved in { -brand-name-firefox } with you everywhere.
 
-# Obsolete string
-navigation-join-firefox = Join { -brand-name-firefox }
-
 navigation-access-all-of-firefox = Access all of { -brand-name-firefox } with a single login — and get more from every product when you do.
 navigation-meet-the-firefox-family = Meet the { -brand-name-firefox } Family
```

Comment:
i think the latter two were supposed to be part of the obsoletes too, being removed around the same period…